executor/executor_test.go

```diff
@@ -1802,15 +1802,15 @@ func (s *testSuiteP1) TestMultiUpdate(c *C) {
 	// Test UPDATE ... set_lists.
 	tk.MustExec(`UPDATE test_mu SET b = 0, c = b WHERE a = 4`)
 	result = tk.MustQuery(`SELECT * FROM test_mu ORDER BY a`)
-	result.Check(testkit.Rows(`1 7 2`, `4 0 0`, `7 8 9`))
```

Comment:
Is the result consistent with MySQL?

Reply:
No. It could be more consistent with PG and SQLite as discussed in #19137.